services/topology-engine/queue-engine/topologylistener/messageclasses.py

```diff
@@ -238,7 +238,7 @@ def handle_link_props(self):
                 raise exc.NotImplementedError(
                     'link props request {}'.format(self.get_message_type()))
         except CypherSyntaxError as e:
-            logger.exception('Invalid request: ', e)
+            logger.exception('Request produce invalid cypher query')
```

Comment:
do we hide Cypher exception here and do not have traceback?

Reply:
according to docs https://docs.python.org/2/library/logging.html#logging.Logger.exception we will have all exception data and traceback